### PR TITLE
add option for "Install all input-plugins"

### PR DESCRIPTION
**Pull request by kkdd**
option "with-all-input-plugins", "Install all input-plugins"

### CODE REVIEW

File: Library/Formula/mapnik.rb [hunk 1]
```diff
@@ -68,6 +70,10 @@ def install
             "CPP_TESTS=False" # too long to compile to be worth it
            ]
 
+     if build.with? "all-input-plugins""
+      args << "INPUT_PLUGINS=all"
+     end
```
Comment (bfontaine) [inline Library/Formula/mapnik.rb]: You can write this on one line:

```ruby
args << "INPUT_PLUGINS=all" if build.with? "all-input-plugins"
```

**Comment by mikemcquaid** [comment]
Any downside to installing these by default?

**Comment by kkdd** [comment]
I can't find any side effects when installing all input-plugins of mapnik. It seems OK for installing these by default.

**Comment by mikemcquaid** [comment]
@kkdd Let's just enable it by default then, thanks!

**Comment by mikemcquaid** [comment]
(and remove the `option` entirely)

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/9e6c76196056a09c424f0626e023c877b8e0ecde. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:.

Just a nit, but non-version formula changes should be styled along the lines of `mapnik: add extra dolphins` or similar, Thanks!

**Pull request closed without merging**